examples/aerial_authz.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2021 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+from datetime import datetime, timedelta
+
+from google.protobuf import any_pb2, timestamp_pb2
+
+from cosmpy.aerial.client import LedgerClient, NetworkConfig
+from cosmpy.aerial.client.utils import prepare_and_broadcast_basic_transaction
+from cosmpy.aerial.tx import Transaction
+from cosmpy.aerial.wallet import LocalWallet
+from cosmpy.crypto.keypairs import PrivateKey
+from cosmpy.protos.cosmos.authz.v1beta1.authz_pb2 import GenericAuthorization, Grant
+from cosmpy.protos.cosmos.authz.v1beta1.tx_pb2 import MsgGrant
+
+
+def main():
+
+    wallet = LocalWallet(PrivateKey("F7w1yHq1QIcQiSqV27YSwk+i1i+Y4JMKhkpawCQIh6s="))
+
+    # Set authz_wallet that will be granted authorization to send tokens from wallet
+    authz_wallet = LocalWallet(
+        PrivateKey("KI5AZQcr+FNl2usnSIQYpXsGWvBxKLRDkieUNIvMOV8=")
+    )
```

Comment:
We don't really need the private key here, just the address. It might even make sense to take this address and `total_authz_time` as command line arguments, similar to how some of the other examples work.

Reply:
But when signing transactions we do need the private key don't we?